sensorsafrica/settings.py

```diff
@@ -109,8 +109,16 @@
     "SENSORSAFRICA_DATABASE_URL",
     "postgres://sensorsafrica:sensorsafrica@localhost:5432/sensorsafrica",
 )
-DATABASES = {"default": dj_database_url.parse(DATABASE_URL)}
+SENSORSAFRICA_READ_DATABASE_URL = os.getenv(
```

Comment:
```suggestion
READ_DATABASE_URL = os.getenv(
```

env name should include `SENSORSAFRICA_` prefix but the setting shouldn’t since they are all for sensors.